app/src/main/java/org/schabi/newpipe/util/external_communication/InternalUrlsHandler.java

```diff
@@ -146,9 +154,15 @@ public static boolean playOnPopup(final Context context,
                 .observeOn(AndroidSchedulers.mainThread())
                 .subscribe(info -> {
                     final PlayQueue playQueue
-                            = new SinglePlayQueue(info, seconds * 1000);
+                            = new SinglePlayQueue(info, seconds * 1000L);
                     NavigationHelper.playOnPopupPlayer(context, playQueue, false);
-                }));
+                }, throwable -> new AlertDialog.Builder(context)
```

Comment:
Idea: Maybe also log the error here so that we have more information in case of debugging

Reply:
Done